statsmodels/_test_runner.py

```diff
@@ -0,0 +1,28 @@
+import os
+import sys
+from distutils.version import LooseVersion
+
+
+class PytestTester(object):
```

Comment:
We already have both tools.testing and tools._testing.  Can't this go in one of those?

Reply:
Some care is needed to avoid circular imports, so maybe